### PR TITLE
Fix broken links in deCONZ

### PR DESCRIPTION
**Pull request by scheric**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/deconz.markdown [hunk 1]
```diff
@@ -35,7 +35,7 @@ ha_platforms:
 
 [deCONZ](https://www.dresden-elektronik.de/funk/software/deconz.html) by [dresden elektronik](https://www.dresden-elektronik.de) is a software that communicates with ConBee/RaspBee Zigbee gateways and exposes Zigbee devices that are connected to the gateway.
 
-[deCONZ REST API](https://dresden-elektronik.github.io/deconz-rest-doc/).
+[deCONZ REST API](https://dresden-elektronik.github.io/deconz-rest-doc/about_rest/).
```
Comment (scheric) [inline source/_integrations/deconz.markdown]: Is this link misplaced here? It does not have any context around it.
```suggestion

```

Comment (frenck) [reply source/_integrations/deconz.markdown]: I agree it is misplaced, let's remove it from here.

Comment (frenck) [reply source/_integrations/deconz.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks, @scheric 👍 

Happy Hacktoberfest 🎉

**Pull request merged**